### PR TITLE
Completed es6-classes exercise - Dulce Herrera

### PR DESCRIPTION
**Pull request by dulceherrera**
Screen Shot 2022-08-29 at 5 38 01 PM
Screen Shot 2022-08-29 at 5 44 04 PM

### CODE REVIEW

File: es6-classes/example-class.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+function ExampleClass() {
+
+}
```
Comment (scbowler) [inline es6-classes/example-class.js]: This should be an empty `class`.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just one small change. Let me know if you have any questions.

File: es6-classes/example-class.js (hunk 1 above)
Comment (dulceherrera) [reply es6-classes/example-class.js]: Screen Shot 2022-08-30 at 11 25 01 AM

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**